src/microbit/__model/display.py

```diff
@@ -4,7 +4,6 @@
 
 from . import constants as CONSTANTS
 from .image import Image
-from .. import shim
 
 
 class Display:

```

Comment:
In the `Display`  and `Button` classes we link to github, while in the `Image` class we link to the docs. Do you think we can make them consistent in this PR? I think the docs make more sense, since it's probably easier to make the version obvious to the reader, but up to you!

Reply:
Ah ok! I'll change it in the Image class! :)